### PR TITLE
Flag to enable railtie implemented

### PR DESCRIPTION
**Pull request by emaglio**
Ciao Nick,
introduced flag to enable/disable `trb-railtie`:
`config.trailblazer.trb_railtie`

So `load_concepts` will be called only if this flag is set to `:enabled` which is the default value.
No test coverage on this but I guess it's quite straight forward.

yew

### CODE REVIEW

File: lib/trailblazer/rails/railtie.rb [hunk 1]
```diff
@@ -5,6 +5,7 @@ module Trailblazer
   class Railtie < ::Rails::Railtie
     config.trailblazer = ActiveSupport::OrderedOptions.new
     config.trailblazer.application_controller ||= :ApplicationController
+    config.trailblazer.trb_railtie ||= :enabled
```
Comment (apotonick) [inline lib/trailblazer/rails/railtie.rb]: Maybe `.use_loader`?

Comment (emaglio) [reply lib/trailblazer/rails/railtie.rb]: OK renamed

**Comment by jagthedrummer** [comment]
Looks like a good start. Where/how would I disable the loader? I'm guessing maybe in `config/application.rb'? Something like:

```ruby
module MyApp
  class Application < Rails::Application
    config.trailblazer.use_loader = :disabled
  end
end
```

The use of a symbol instead of a boolean for this strikes me as odd. It seems like there would only ever be two states ("yes, use the loader" or "no, don't") so I'd instinctively expect to be able to do:

```ruby
config.trailblazer.use_loader = false
```

or 

```ruby
config.trailblazer.use_loader = true
```

**Comment by emaglio** [comment]
@jagthedrummer yeah in the `application.rb` should be good, I have put it in an initializer to test in one of my app, anyway that won't change much.

Regarding the boolean flag, totally agreed! I had something different in mind and then I haven't changed it....now it looks better 3206a1d2a37f426912d7bd1c06438502e620d7cc

**Comment by apotonick** [comment]
Looks good to me. Should we make the `loader` dependency in the gemspec optional?

**Comment by apotonick** [comment]
@seuros Do we need the `Gemfile.lock`?

**Comment by seuros** [comment]
We need to rebase this branch

**Comment by emaglio** [comment]
@seuros which branch you need to rebase from? just because it is already rebase with `master`

**Comment by apotonick** [comment]
@seuros can you cherry-pick the commits into the 2.0 branch, please?

**Comment by apotonick** [comment]
Tested it, excellent work, @emaglio. :ok_hand: 

I actually find that, with the `workflow` gem that gives you a workflow (where you barely call an operation itself), it's ok to name things the Rails Way as the `Operation::` doesn't hurt too much.

**Pull request merged**